GOVERNANCE.md

```diff
@@ -0,0 +1,140 @@
+Anyone may contribute, and contributions are available to all users according to open source values and licenses.
+- **Transparent:**
+Flux strives for transparency in all discussions, announcements, disclosures and decision making.
+- **Unbiased:**
+Flux strives to operate independently of specific partisan interests, and for decision making to fairly balance the wider community interests of its end users and contributors.
+
+### Code of Conduct
+
+The Flux community adheres to the CNCF Code of Conduct <https://github.com/cncf/foundation/blob/master/code-of-conduct.md>.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a _Flux_ Oversight Committee member.
+
+If no conclusion can be reached in meditation, such issues can be escalated to the CNCF mediator, Mishi Choudhary <mishi@linux.com>, in which case CNCF may choose to intervene.
+
+## Roles in the Flux Community
+
+The Flux community comprises the following roles:
+
+### Users
+
+Flux end users are the most important aspect of the community, without whom the project would have no purpose. Users are anyone who has a need for the project.
+Apart from following the Code of Conduct, there are no special requirements.
+
+### Contributors
+
+Flux welcomes all kinds of contributions, including code, issues, documentation, external tools, advocacy and community work.
+As a contributor we want to invite you to join the discussions in a variety of forums laid out in <https://github.com/fluxcd/community>.
+
+### Maintainers
+
+Maintainers are elected Contributors who showed significant and sustained contributions in a git repository.
+Current Maintainers are listed in a `MAINTAINERS` file at the root of the git repository.
+
+Maintainers are expected to:
+
+- Enable and promote Flux community values
+- Engage with end Users through appropriate communication channels
+- Serve as a point of conflict resolution between Contributors to their git repository
+- Maintain open collaboration within and across teams
+- Ask for help when unsure and step down considerately
+
+Maintainers will be invited to the `@fluxcd/maintainers` <https://github.com/orgs/fluxcd/teams/maintainers> team, and are members of this team for as long as they are involved with the project.
+
+### Oversight Committee
+
+This committee is responsible for the overall project, and anything not easily managed by the Maintainers of each git repository. Including:
+
+- Overseeing the project health and growth
+- Maintaining the brand, mission, vision, values, and scope of the overall project
+- Changes to licensing and intellectual property
+- Administering access to all project assets
+- Administering git repositories as needed
+- Handling Code of Conduct violations
+- Managing financial decisions
+- Defining the scope of each git repository
+- Resolving escalated decisions when the git repository Maintainers responsible are blocked
```

Comment:
```suggestion
- Resolving escalated decisions when the Maintainers responsible are blocked
```

I think mentioning "git repository" here is redundant.

Reply:
Or even just:

```suggestion
- Resolving escalated decisions when Maintainers responsible are blocked
```